apis/placement/v1alpha1/disruptionbudget_types.go

```diff
@@ -65,8 +66,6 @@ type PlacementDisruptionBudgetSpec struct {
 	// This field is mutually exclusive with the MinAvailable field in the spec; exactly one
 	// of them can be set at a time.
 	//
-	// Defaults to 25%.
```

Comment:
Error returned when both MaxUnavailable and MinAvailable aren't specified,

The ClusterResourcePlacementDisruptionBudget "test-crpdb" is invalid: 
* spec: Required value
* <nil>: Invalid value: "null": some validation rules were not checked because the object was invalid; correct the existing errors to complete validation